src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -50,120 +50,130 @@ private static void InitSentryAndroidSdk(SentryOptions options)
+            o.AnrTimeoutIntervalMillis = (long)options.Android.AnrTimeoutInterval.TotalMilliseconds;
+            o.EnableActivityLifecycleBreadcrumbs = options.Android.EnableActivityLifecycleBreadcrumbs;
+            o.EnableAutoActivityLifecycleTracing = options.Android.EnableAutoActivityLifecycleTracing;
+            o.EnableActivityLifecycleTracingAutoFinish = options.Android.EnableActivityLifecycleTracingAutoFinish;
+            o.EnableAppComponentBreadcrumbs = options.Android.EnableAppComponentBreadcrumbs;
+            o.EnableAppLifecycleBreadcrumbs = options.Android.EnableAppLifecycleBreadcrumbs;
+            o.EnableSystemEventBreadcrumbs = options.Android.EnableSystemEventBreadcrumbs;
+            o.EnableUserInteractionBreadcrumbs = options.Android.EnableUserInteractionBreadcrumbs;
+            o.EnableUserInteractionTracing = options.Android.EnableUserInteractionTracing;
+
+            // These options are in Java.SentryOptions but not ours
+            o.AttachThreads = options.Android.AttachThreads;
+            o.ConnectionTimeoutMillis = (int)options.Android.ConnectionTimeout.TotalMilliseconds;
+            o.EnableNdk = options.Android.EnableNdk;
+            o.EnableShutdownHook = options.Android.EnableShutdownHook;
+            o.EnableUncaughtExceptionHandler = options.Android.EnableUncaughtExceptionHandler;
+            o.ProfilesSampleRate = (JavaDouble?)options.Android.ProfilesSampleRate;
+            o.PrintUncaughtStackTrace = options.Android.PrintUncaughtStackTrace;
+            o.ReadTimeoutMillis = (int)options.Android.ReadTimeout.TotalMilliseconds;
+
+            // In-App Excludes and Includes to be passed to the Android SDK
+            options.Android.InAppExcludes?.ForEach(o.AddInAppExclude);
+            options.Android.InAppIncludes?.ForEach(o.AddInAppInclude);
+
+            // These options are intentionally set and not exposed for modification
+            o.EnableExternalConfiguration = false;
+            o.EnableDeduplication = false;
+            o.AttachServerName = false;
+
+            // These options are intentionally not expose or modified
+            //o.MaxRequestBodySize   // N/A for Android apps
+            //o.MaxSpans             // See https://github.com/getsentry/sentry-dotnet/discussions/1698
+
+            // Don't capture managed exceptions in the native SDK, since we already capture them in the managed SDK
+            o.AddIgnoredExceptionForType(JavaClass.ForName("android.runtime.JavaProxyThrowable"));
+        });
+
+        // Now initialize the Android SDK (with a logger only if we're debugging)
+        if (options.Debug && options.DiagnosticLogger is {} logger)
+        {
+            var androidLogger = new AndroidDiagnosticLogger(logger);
+            SentryAndroid.Init(AppContext, androidLogger, configuration);
+        }
+        else
+        {
+            SentryAndroid.Init(AppContext, configuration);
+        }
```

Comment:
This part changed - we used to always pass in a logger.  Now we only pass one in when debugging.